podcasts/UpNextViewController.swift

```diff
@@ -221,10 +219,47 @@ class UpNextViewController: UIViewController, UIGestureRecognizerDelegate {
     }
 
     @objc private func themeDidChange() {
-        let unselected = UIImage(named: "shuffle")?.withTintColor(AppTheme.colorForStyle(.primaryIcon02, themeOverride: themeOverride), renderingMode: .alwaysOriginal)
-        let selected = UIImage(named: "shuffle-enabled")?.withTintColor(AppTheme.colorForStyle(.primaryIcon01, themeOverride: themeOverride), renderingMode: .alwaysOriginal)
-        shuffleButton.setImage(unselected, for: .normal)
-        shuffleButton.setImage(selected, for: .selected)
+        if !SubscriptionHelper.hasActiveSubscription() || !SyncManager.isUserLoggedIn() {
+            shuffleButton.setImage(UIImage(named: "shuffle-plus"), for: .normal)
```

Comment:
I believe you also nee to change the `selected` status to the `shuffle-plus` image, or else if it was on the selected state when a subscription was active it will swill stay like that.
Related to this we also should disable shuffle if no subscription is active, because at the moment it will stay active if you had if before with a subscription.

Reply:
Good catch @SergioEstevao ! Ready for another look!